includes/views/advanced-tools.php

```diff
@@ -8,9 +8,11 @@
 			<h4><?php _e( 'Generate random temporary directory', 'woocommerce-pdf-invoices-packing-slips' ); ?></h4>
 			<p><?php _e( 'For security reasons, it is preferable to use a random name for the temporary directory.', 'woocommerce-pdf-invoices-packing-slips' ); ?></p>
 			<form method="post">
-				<?php wp_nonce_field( 'wpo_wcpdf_debug_tools_action', 'security' ); ?>
-				<input type="hidden" name="wpo_wcpdf_debug_tools_action" value="generate_random_string">
-				<input type="submit" name="submit" id="submit" class="button" value="<?php esc_attr_e( 'Generate temporary directory', 'woocommerce-pdf-invoices-packing-slips' ); ?>">
+				<input type="hidden" name="debug_tool" value="generate_random_string">
+				<a href="" class="button button-secondary submit"><?php _e( 'Generate temporary directory', 'woocommerce-pdf-invoices-packing-slips' ); ?></a>
```

Comment:
Maybe using an `<input type="submit">` was better for compatibility and accessibility?